framework/source/class/qx/ui/table/IColumnMenuItem.js

```diff
@@ -21,14 +21,6 @@
  */
```

Comment:
It may be valuable to leave a comment here why we do not use a boolean property visible, but a set of setter and getter.